changelog/fragments/bump-kubebuilder-add-kustomize.yaml

```diff
@@ -0,0 +1,104 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: Legacy layout "Kubebuilder 2.x" provided via the go/v2 plugin is officially deprecated. Be aware that since `Apr 2021`, the default layout produced is done via the `go/v3`. ([More info](https://github.com/kubernetes-sigs/kubebuilder/discussions/2842)) Please, check your PROJECT file and ensure and then, if your project layout is not `go.kubebuilder.io/v3` follow the [migration guide](https://sdk.operatorframework.io/docs/building-operators/golang/migration/) to upgrade your project. 
```

Comment:
```suggestion
  - description: Legacy layout "Kubebuilder 2.x" provided via the go/v2 plugin is officially deprecated. Be aware that since `Apr 2021`, the default layout produced is done via the `go/v3`. ([More info](https://github.com/kubernetes-sigs/kubebuilder/discussions/2842)) Check your `PROJECT` file to make sure that your project layout is `go.kubebuilder.io/v3`, if not please follow the [migration guide](https://sdk.operatorframework.io/docs/building-operators/golang/migration/) to upgrade your project. 
```